other/appdata/antimicroX.appdata.xml.in

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <component type="desktop">
-  <id>com.github.juliagoda.antimicroX.desktop</id>
+  <id>antimicroX.desktop</id>
```

Comment:
It was `com.github.juliagoda.antimicroX` because this was the unique Appstream ID for the app.
Specs:
https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

> The ID must follow a reverse-DNS scheme, consisting of {tld}.{vendor}.{product}, for example org.kde.gwenview or com.hugski.colorhug2. Ownership of {vendor}.{tld} in the domain name system guarantees uniqueness of IDs. 

> Note that the value of this tag must be unique...

Although it passes `appstream-util validate-strict --nonet ./antimicroX.appdata.xml` ([here](https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/#_app_data_validate_usage)), I suggest `com.github.AntiMicroX.antimicroX` for uniqueness. Same for relevant file names.

Reply:
Thanks, I was not aware of this standard.

I understand an ID is just a regular id, but in this case we can see here name for `.desktop` file.
Is this ID just name for every icon and .desktop file for an app?

Or maybe `com.github.juliagoda.antimicroX.desktop` was wrong and in this place `com.github.juliagoda.antimicroX` should be placed?

Reply:
> Is this ID just name for every icon and .desktop file for an app?

Indeed, see [here](https://docs.fedoraproject.org/en-US/flatpak/concepts/#application_id). This will be required for flatpak, too:

> All resources exported by the application must be prefixed by this identifier. This includes the desktop file, the appdata file for the application, and any icons referenced by the desktop file.

The appdata file `<component type="desktop">` refers to a desktop application.
Because we are choosing a new ID, we can drop the `.desktop` part, see [here](https://www.freedesktop.org/software/appstream/docs/sect-Metadata-Application.html#tag-id-desktopapp):

> In modern metainfo files, the component-ID for desktop-application components can be an arbitrary reverse-DNS string (matching the naming rules applying to all AppStream metadata), while the <launchable/> tag is used to associate .desktop files with their metainfo files. 

So you only need `.desktop` in the `<launchable>` tag.

Before Flathub uppload I'll [review](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps) the appstream files.

Reply:
> it is strongly encouraged to only use lowercase letters for the ID.

I have changed our id to `com.github.antimicrox.antimicrox.desktop`